src/components/containers/fieldset/Fieldset.specs.js

```diff
@@ -9,7 +9,10 @@ import Fieldset from './Fieldset';
 describe('Fieldset component', () => {
   it('adds additional classes to legend element when aditionalLegendClasses prop is supplied', () => {
     const instance = mount(
-      <Fieldset legendText="I am legend" additionalLegendClasses="legendary" />
+      <Fieldset
+        legendContent="I am legend"
+        additionalLegendClasses="legendary"
```

Comment:
I think `legendClasses` or `legendClassname` would suffice, partly because I want an excuse to misread this as `legendaryClasses` but mostly because there really isn't anything "additional" about the prop.